gwt/src/main/resources/BUILD

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2017 The Dagger Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description:
+#   Dagger GWT integration
+
+package(default_visibility = ["//:src"])
+
+java_library(
+    name = "gwt",
+    resources = glob(["**/*.gwt.xml"]),
+    exports = [
+        "//core/src/main/java/dagger:core",
```

Comment:
What do you think about specifying the `sources` classifier in a tag? Then this becomes general purpose, plus youc can have some deps that have classifiers and others that don't?